app/services/retention_schedules/add_schedule_service.rb

```diff
@@ -90,10 +90,12 @@ def planned_destruction_date
     end
 
     def closure_date
-      if @kase.last_transitioned_at.nil?
-        @kase.transitions.most_recent.created_at.to_date
+      # all cases recieve a date 
```

Comment:
Super minor thing, but the comment feels misleading as it says all cases receive a date responded on closure, but there is a conditional in case it doesn't (which you mention in the PR can happen with some old cases) 😄

Reply:
haha yes this doesn't make sense now, I wrote it when i thought it was the case and then forgot to do update it 🤦 

A micro lesson in why comments are not to be relied on!